### PR TITLE
fix(svg-icons): default icons color

### PR DESCRIPTION
**Pull request by amtins**
## Description

Uses the same color as defined by the `color` property of the `video-js` class to apply to `svg icons` for easy customization.

```css
.video-js {
  color: red; /* Also redefines svg icons color  */
}
```

## Specific Changes proposed

- set `fill` property to `currentColor`

## Requirements Checklist
- [X] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/css/_icons.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
   background-repeat: no-repeat;
   background-position: center;
 
-  fill: #FFFFFF;
+  fill: currentColor;
```
Comment (wseymour15) [inline src/css/_icons.scss:14]: I could be wrong here and I haven't tested, but should this be [currentcolor](https://www.w3schools.com/colors/colors_currentcolor.asp)?

Comment (gkatsev) [reply src/css/_icons.scss:14]: Looks like it https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword
though, I think it's case-insensitive

Comment (amtins) [reply src/css/_icons.scss:14]: @wseymour15, @gkatsev thanks for the review! 

Would you like me to adjust the value ? If it helps to make a decision, here's what the W3C says https://www.w3.org/TR/css-color-3/#currentcolor

Comment (wseymour15) [reply src/css/_icons.scss:14]: It's always fun when docs conflict 😄 You can leave as is. No strong feelings after reading that.

Comment (gkatsev) [reply src/css/_icons.scss:14]: yeah, interesting that mdn doesn't document it like the spec. I would always err on the side of the spec, if possible.

**Review by dzianis-dashkevich — APPROVED** [review]

**Pull request merged**